### DISCUSSION TITLE
[09 Settembre 2022] - Cache

### DISCUSSION THREAD

**Discussion by alem1105** [CS-Swap/Architettura-degli-Elaboratori#15] [Cache]
### Traccia Esercizio

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/a9689da8-c900-4081-aae3-0bb1ceee438e)

<details><summary>Soluzione Prof</summary>
<p>

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/7258e0c4-1e7d-4843-ada5-6ccc3527b63a)

</p>
</details> 

---

>**[[Altre Soluzioni]](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/60)**

**Comment by thegeek-sys** [comment]
<img width="1083" alt="Screenshot 2024-06-03 alle 21 54 48" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/327be217-d1a8-4201-bce4-2d8e8135de41">

**Comment by rimaout** [comment]
### 1)
<img width="1022" alt="90C47210-9D99-4C03-838E-D6B663E13728" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/2059f310-c433-4ecb-97cb-25308489fc5d">

### 2)
<img width="1042" alt="44BCAF6C-6B19-4809-876F-9BE11990B70F" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/e12d1463-9efd-4e78-81a0-2ab85a17d053">

### 3)
![IMG_3152](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/90c3cfaf-4f55-44f2-95ce-7aa1331ffa7b)

### 4)
<img width="948" alt="F8A3734E-AE66-4FFE-8C12-3F98F6DD202B" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/125dd982-abe9-4493-91c3-cb1e449b92d9">

### 5)
<img width="853" alt="0D2D9455-61FA-47BE-B332-A7A5C420C95E" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/013f315a-b27c-4799-9de4-6a0a134a4acd">

**Comment by Divine-Sunderer** [comment]
<img width="3965" alt="09 Settembre 2022  - Cache" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/157913185/d1f48688-bb4d-4385-80f9-2e9534dd7d18">
